src/freenas/usr/local/lib/middlewared_truenas/plugins/truenas.py

```diff
@@ -44,6 +44,9 @@ class TruenasCustomerInformationModel(sa.Model):
 
 class TrueNASService(Service):
 
+    class Config:
+        cli_namespace = "system.truenas"
```

Comment:
I dont see this service being useful in CLI at all

Reply:
Won't `accept_eula`, `update_customer_information` and `set_production` methods be useful for the support team?

Reply:
Not really, maybe the last two only.

There is a number that would not make much sense as a CLI interface, e.g. get_chassis_hardware, get_eula, etc